src/test/java/de/mediathekview/mserver/crawler/wdr/parser/WdrFilmDeserializerTest.java

```diff
@@ -0,0 +1,277 @@
+            "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/158/1583693/,1583693_18232361,1583693_18232358,1583693_18232363,1583693_18232362,1583693_18232359,1583693_18232360,.mp4.csmil/index_4_av.m3u8",
+            "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/158/1583693/,1583693_18232391,1583693_18232389,1583693_18232393,1583693_18232392,1583693_18232388,1583693_18232390,.mp4.csmil/index_0_av.m3u8",
+            "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/158/1583693/,1583693_18232391,1583693_18232389,1583693_18232393,1583693_18232392,1583693_18232388,1583693_18232390,.mp4.csmil/index_2_av.m3u8",
+            "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/158/1583693/,1583693_18232391,1583693_18232389,1583693_18232393,1583693_18232392,1583693_18232388,1583693_18232390,.mp4.csmil/index_4_av.m3u8",
+            "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/158/1583693/,1583693_18232344,1583693_18232347,1583693_18232346,1583693_18232345,1583693_18232343,1583693_18232342,.mp4.csmil/index_0_av.m3u8",
+            "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/158/1583693/,1583693_18232344,1583693_18232347,1583693_18232346,1583693_18232345,1583693_18232343,1583693_18232342,.mp4.csmil/index_2_av.m3u8",
+            "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/158/1583693/,1583693_18232344,1583693_18232347,1583693_18232346,1583693_18232345,1583693_18232343,1583693_18232342,.mp4.csmil/index_4_av.m3u8",
+            new GeoLocations[] {GeoLocations.GEO_NONE}
+          }
+        });
+  }
+
+  private final String requestUrl;
+  private final String filmPageFile;
+  private final String jsUrl;
+  private final String jsFile;
+  private final String m3u8UrlVideo;
+  private final String m3u8FileVideo;
+  private final String m3u8UrlSignLanguage;
+  private final String m3u8FileSignLanguage;
+  private final String m3u8UrlAudioDescription;
+  private final String m3u8FileAudioDescription;
+  private final String topic;
+  private final String expectedTitle;
+  private final LocalDateTime expectedDate;
+  private final Duration expectedDuration;
+  private final String expectedDescription;
+  private final String expectedSubtitle;
+  private final String expectedUrlSmall;
+  private final String expectedUrlNormal;
+  private final String expectedUrlHd;
+  private final String expectedUrlSignLanguageSmall;
+  private final String expectedUrlSignLanguageNormal;
+  private final String expectedUrlSignLanguageHd;
+  private final String expectedUrlAudioDescriptionSmall;
+  private final String expectedUrlAudioDescriptionNormal;
+  private final String expectedUrlAudioDescriptionHd;
+  private final GeoLocations[] expectedGeoLocations;
+
+  public WdrFilmDeserializerTest(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 26 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)